File: app/src/main/java/com/adonai/manman/preferences/PreferencesActivity.java
package com.adonai.manman.preferences;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import com.adonai.manman.R;
import com.adonai.manman.Utils;

/**
 * Activity for holding and showing preference fragments
 *
 * @author Oleg Chernovskiy
 */
public class PreferencesActivity extends AppCompatActivity {

    private Toolbar mActionBar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        // should set theme prior to instantiating compat actionbar etc.
        Utils.setupTheme(this);
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_preferences);

        mActionBar = (Toolbar) findViewById(R.id.app_toolbar);
        setSupportActionBar(mActionBar);

        getFragmentManager().beginTransaction().replace(R.id.content_frame, new GlobalPrefsFragment()).commit();
    }

}

File: app/src/main/java/com/adonai/manman/preferences/PreferencesActivity.kt
package com.adonai.manman.preferences

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar
import com.adonai.manman.R
import com.adonai.manman.Utils

/**
 * Activity for holding and showing preference fragments
 *
 * @author Kanedias
 */
class PreferencesActivity : AppCompatActivity() {

    private lateinit var mActionBar: Toolbar

    override fun onCreate(savedInstanceState: Bundle?) {
        // should set theme prior to instantiating compat actionbar etc.
        Utils.setupTheme(this)
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_preferences)

        mActionBar = findViewById<View>(R.id.app_toolbar) as Toolbar
        setSupportActionBar(mActionBar)

        supportFragmentManager.beginTransaction().replace(R.id.content_frame, GlobalPrefsFragment()).commit()
    }
}